opal/mca/pmix/pmix1xx/pmix/include/private/autogen/config.h.in

```diff
@@ -0,0 +1,636 @@
+/* include/private/autogen/config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
This is an automatically generated file that should not be part of this commit.
assuming you branched from a recent master, there should not be a opal/mca/pmix/pmix1xx directory in your source tree, and you can safely remove it.